src/config.rs

```diff
@@ -133,6 +138,18 @@ impl Config {
             config.excerpt_separator = excerpt_separator.to_owned();
         };
 
+        #[cfg(all(feature="syntax-highlight", not(windows)))]
+        {
+            if let Some(theme) = yaml["syntax-highlight"]["theme"].as_str() {
+                if has_theme(theme) {
+                    config.theme = theme.to_owned();
+                } else {
+                    warn!("Theme named '{}' is not loaded. Using default 'base16-ocean.dark'.",
+                          theme);
+                }
+            };
+        }
+
```

Comment:
I think i'd rather isolate the feature knowledge by having this run unconditionally.

In thinking of the case where someone is developing a blog on windows and deploying with TravisCI, it'd be nice if `has_theme` unconditionally returned true.  This is a bit of a lie but then again, no validation is performed otherwise because of the feature toggle.